components/table/ProductTableRow.tsx

```diff
@@ -94,8 +94,8 @@ const ProductTableRow = ({ product }: Props) => {
         <div data-testid='product-last-update'>
           <p className='product-last-update'>{productLastUpdate}</p>
         </div>
-        <div>
-          <p></p>
+        <div data-testid='product-overall-compatibility'>
+          <p>{`${product.overallCompatibility === 1 ? '100' : (product.overallCompatibility*100).toFixed(2)}%`}</p>
```

Comment:
I think we can just keep integer for all values not only 100. I know that we have more data coming from the backend, but we should be consistent with mocks.